templates/BlueGrey/index.tmpl

```diff
@@ -55,14 +55,6 @@
   [websvn-endtest]
     </table>
   [websvn-test:treeview]
-    <script type="text/javascript" src="[websvn:locwebsvnhttp]/javascript/group-collapse.js"></script>
-    [websvn-test:opentree]
```

Comment:
Are you sure removing those parts is correct? I don't see any other implementation of that behaviour in JS. Looks to me like this needs to be reintroduced.

Reply:
No No no this was not correct i tried to move this ope-tree selection in JS but couldnt succeed. So I have reverted it back. This i will take care in separate issue. For now i have reverted it back with collapsible.js and added the call of jquery-min version 1.9.1 file on top of this line.